example/04_gemm_add_add_fastgelu/CMakeLists.txt

```diff
@@ -1 +1,4 @@
+add_example_executable(example_gemm_add_add_fastgelu_xdl_bf16 gemm_add_add_fastgelu_xdl_bf16.cpp)
```

Comment:
I am curious is verification pass on bf16?
In the past, we verify bf16(gpu) and fp32(cpu).
Because we have using bhalf_t = ushort; instead of real floating representation.
I found some initialization of gemm_xdl_bf16 might wrong.
So, I am curious is it work now?

Reply:
previosuly, we wer using ```static_cast``` to convert between bf16 and fp32. Now use ```type_convert``` which will convert correctly